tests/series/test_size_bytes.py

```diff
@@ -88,6 +88,24 @@ def test_series_date_size_bytes(size, with_nulls) -> None:
     assert s.size_bytes() == get_total_buffer_size(data)
 
 
+@pytest.mark.parametrize("size", [0, 1, 2, 8, 9, 16])
+@pytest.mark.parametrize("with_nulls", [True, False])
+def test_series_time_size_bytes(size, with_nulls) -> None:
```

Comment:
Might be good to have this test (and maybe some others) parametrized by the time unit, to ensure that the nanosecond resolution works/is covered as well.